ledger/catchpointtracker.go

```diff
@@ -354,13 +354,15 @@ func (ct *catchpointTracker) loadFromDisk(l ledgerForTracker, dbRound basics.Rou
 		return err
 	}
 
+	ct.catchpointsMu.Lock()
 	ct.cachedDBRound = dbRound
 	ct.roundDigest = nil
 	ct.consensusVersion = nil
 	ct.catchpointDataWriting.Store(0)
 	// keep these channel closed if we're not generating catchpoint
 	ct.catchpointDataSlowWriting = make(chan struct{}, 1)
 	close(ct.catchpointDataSlowWriting)
+	ct.catchpointsMu.Unlock()
```

Comment:
Can this close panic by any chance/is there a risk of us not unlocking?

Reply:
closing right after initializing under a lock, what can go wrong?